de/deploy/README.md

```diff
@@ -26,9 +26,9 @@ Git verwaltet die Veränderungen an einer Sammlung von Dateien in einem sogenann
 
     $ git init
     Initialized empty Git repository in ~/djangogirls/.git/
-    $ git config --global user.name "Dein Name"
-    $ git config --global user.email du@beispiel.com
```

Comment:
Now changed on Crowdin

Reply:
Most probably it was made manually. Every time we update English version every other language coverage drop. It occurs that sometimes Crowdin knows, that translation, but because `XPath` of sentence to be translated is changed, it drops its translation.

So I just walkthrough few target languages and re-introduced these translations (it there were 100% match). Sorry for that!